crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -290,20 +308,50 @@ module bridge::committee {
         abort ESenderIsNotInBridgeCommittee
     }
 
+    public(package) fun update_bridge_network_pubkey(
```

Comment:
do we allow validator to update the network pub key after registration? we will need to expose this function in bridge.move if we want to support that.